.github/workflows/flutter-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: '12.x'
-      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.5 _flutter
+      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.6 _flutter
```

Comment:
Doesn't seem like a good way to share parameters across yaml files, so don't worry about it